packages/app/src/systems/FundWallet/hooks/useFundWallet.tsx

```diff
@@ -0,0 +1,50 @@
+import { Address } from 'fuels';
+import { useCallback, useMemo } from 'react';
+import { IS_CRX } from '~/config';
+import { DEFAULT_NETWORKS } from '~/networks';
+import { useAccounts } from '~/systems/Account';
+import { openTab } from '~/systems/CRX/utils';
+import { useNetworks } from '~/systems/Network';
+
+export function useFundWallet() {
+  const { selectedNetwork } = useNetworks();
+  const { account } = useAccounts();
+
+  const { faucetUrl, bridgeUrl } = useMemo(() => {
+    const network = DEFAULT_NETWORKS.find(
+      (n) => n.url === selectedNetwork?.url
```

Comment:
Shouldn't the selected network obj come ready to use from useNetworks/NetworkMachine?

Reply:
It dosen't have the bridge or facuet url, and I don't think we should add to the database.